Dockerfile_dev

```diff
@@ -6,7 +6,7 @@ ENV UWSGI_HTTP_AUTO_CHUNKED=1 UWSGI_HTTP_KEEPALIVE=1 UWSGI_LAZY_APPS=1 UWSGI_WSG
 ## System preparation steps ################################################# ##
 
 # !!! Runtime changes won't affect requirements.txt
-COPY requirements.txt requirements_test.txt /tmp/
+COPY requirements.txt requirements_test.txt requirements_k8s.txt /tmp/
```

Comment:
You also need to change the filename below:

`RUN zypper -n in python310-devel gcc libffi-devel aws-cli && pip3.10 install -r /tmp/requirements_test.txt && rm -rf /var/cache`